editor/src/components/canvas/design-panel-root.tsx

```diff
@@ -300,25 +300,26 @@ const DesignPanelRootInner = React.memo(() => {
             {isCanvasVisible && navigatorVisible ? (
               <div
                 style={{
-                  height: '100%',
+                  height: '90%',
                   position: 'absolute',
                   top: 0,
                   left: 0,
-                  zIndex: 20,
-                  overflow: 'hidden',
-                  borderLeft: `1px solid ${colorTheme.subduedBorder.value}`,
-                  borderRight: `1px solid ${colorTheme.subduedBorder.value}`,
+                  zIndex: 1,
```

Comment:
do you even still need this?

Reply:
the zIndex yes, the navigator isn't visible without it. I'm assuming it goes beneath the canvas. The things that need to float above it like the delete file dialog is still good